testing_wasm/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "testing",
+  "version": "1.0.0",
+  "description": "Testing",
+  "scripts": {
+    "start": "node -r dotenv/config index.js",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "Utkarsh Varma",
+  "dependencies": {
+    "@cosmjs/stargate": "^0.28.4",
+    "@cosmjs/cosmwasm-stargate": "^0.28.4",
+    "@cosmjs/proto-signing": "^0.28.4",
+    "chai": "^4.3.6",
+    "node-fetch": "^3.2.5",
+    "protobufjs": "^6.10.2"
+  },
+  "devDependencies": {
+    "@terra-money/terra.js": "^2.0.16",
```

Comment:
why do we need this "terra.js" dep?

Reply:
Sorry the testing setup is from an old config I will remove it